src/api/clearlyDefined.js

```diff
@@ -90,6 +91,12 @@ export function curate(token, spec) {
   return patch(url(`${CURATIONS}`), token, spec)
 }
 
+/**
+ * Get details about a specific definition
+ * @param {*} token
+ * @param {*} entity
+ * @param {object} params can contain properties: expandPrs, if true include deeper information for each PR like the title, message
```

Comment:
similar to above, there should be an `expand` arg that takes an array of enum values as to what to include.